Lib/test/test_dis.py

```diff
@@ -257,6 +257,9 @@ def wrap_func_w_kwargs():
               LOAD_CONST               3 (5)
               LOAD_CONST               4 (('c',))
               CALL_KW                  3
+              CACHE
```

Comment:
AFAIK, CACHE is not supposed to show up. Just like LOAD_GLOBAL has no CACHE showing up after. We need to teach dis about CALL_KW.

Reply:
Fixed. `CALL_KW` needed to be added to the cache table in opcode.py